### ISSUE TITLE
梯度反传时SRU的gn.gamma和gn.beta的梯度grad=None

### ISSUE THREAD

**Issue by zhangzh16** [cheng-haha/ScConv#8]
在直接搬过来替换3*3模块的时候发现的这个问题，SRU.gn.gamma和SRU.gn.beta的grad是None，网络其他参数没有问题，想问下这是正常的吗？如果正常的话那这两个参数还会更新么

下面是一段示例代码
if __name__ == '__main__':
    x       = torch.randn(16,32,80,201)
    y = torch.randn(16,32,80,201)
    
    model   = ScConv(32)
    
    loss_fn=nn.MSELoss()
    optimizer=torch.optim.SGD(model.parameters(),lr=0.1)
    
    out=model(x)
    print(model(x).shape)
    loss=loss_fn(out,y)
    
    optimizer.zero_grad()
    loss.backward()
    print('-------------grad is False ARE below--------------------')
    for name, param in model.named_parameters():
        if param.grad is None:
            print(name)
    print('+++++++++++++grad is True ARE below++++++++++++++++++++++')
    for name, param in model.named_parameters():
        if param.grad is not None:
            print(name)

输出结果是
torch.Size([16, 32, 80, 201])
-------------grad is False ARE below--------------------
SRU.gn.gamma
SRU.gn.beta
+++++++++++++grad is True ARE below++++++++++++++++++++++
CRU.squeeze1.weight
CRU.squeeze2.weight
CRU.GWC.weight
CRU.GWC.bias
CRU.PWC1.weight
CRU.PWC2.weight

**Comment by cheng-haha** [comment]
好的，之前改动的断掉一个叶子节点了，抱歉，现已经改正，请重新测试吧
